python-frontend/src/test/java/org/sonar/python/semantic/v2/typeshed/VarSymbolToDescriptorConverterTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * SonarQube Python Plugin
+ * Copyright (C) 2011-2024 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.python.semantic.v2.typeshed;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.sonar.python.index.VariableDescriptor;
+import org.sonar.python.types.protobuf.SymbolsProtos;
+
+class VarSymbolToDescriptorConverterTest {
+
+  @Test
+  void test() {
```

Comment:
As mentioned in the other comment, I guess we'd probably like to have a test for something that is from the `builtins` module. And maybe a test of a "real symbol" that comes from a class, to see if that is the output we want (that latter test might be one that is a bit more functional in nature, and integrated with an actual conversion - maybe the converter needs some additional context...)

Reply:
as discussed - the test with the `ProtoSymbol` data from real proto files will be added as a part of [SONARPY-2160 PR](https://github.com/SonarSource/sonar-python/pull/2005)